pkg/upgrade/upgrader/jiva_upgrade.go

```diff
@@ -633,3 +683,225 @@ func jivaUpgrade(pvName, openebsNamespace string, utaskObj *utask.UpgradeTask) (
 	klog.Info("Upgrade Successful for", pvName)
 	return options.utaskObj, nil
 }
+
+func (j *jivaVolumeOptions) cleanup(openebsNamespace string) error {
+	var err error
+	if j.replicaObj.version == currentVersion && j.replicaObj.name != "" {
```

Comment:
can we also add a check on the namespace here.

Reply:
Namespace check is not required as irrespective of namespace either openebs or pvc, the old deployment `pvName-rep` is deleted after splitting it.